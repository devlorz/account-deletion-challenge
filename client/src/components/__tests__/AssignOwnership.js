import React from 'react'
import { render } from 'react-testing-library'

import AssignOwnership from '../AssignOwnership'

test('snapshot', () => {
  const { container } = render(
    <AssignOwnership
      user={{
        email: 'ross@example.com',
        name: 'Ross Lynch',
        _id: 'user1',
      }}
      transferData={[
        {
          status: 'completed',
          toUser: {
            name: 'Ryan Lynch',
            _id: 'user2',
          },
          workspaceId: 'workspace1',
        },
      ]}
      workspace={{
        displayName: 'Lightning strike',
        spaceId: 'workspace1',
        transferableMembers: [
          {
            name: 'Ryan Lynch',
            _id: 'user2',
          },
          {
            name: 'Riker Lynch',
            _id: 'user3',
          },
          {
            name: 'Rydel Lynch',
            _id: 'user4',
          },
        ],
      }}
      onAssignToUser={jest.fn()}
    />
  )
  expect(container.firstChild).toMatchSnapshot()
})
