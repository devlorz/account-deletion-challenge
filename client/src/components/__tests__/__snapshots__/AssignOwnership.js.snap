// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  data-reactroot=""
  style="text-decoration: underline; cursor: pointer;"
>
  <select
    style="min-width: 3rem;"
  >
    <option
      disabled=""
      value=""
    />
    <option
      value="user2"
    >
      Ryan Lynch
    </option>
    <option
      value="user3"
    >
      Riker Lynch
    </option>
    <option
      value="user4"
    >
      Rydel Lynch
    </option>
  </select>
</div>
`;
